front/components/assistant_builder/actions/configuration/ChildAgentConfigurationSection.tsx

```diff
@@ -128,7 +130,9 @@ export function ChildAgentConfigurationSection({
           <div className="flex h-full w-full items-center justify-center">
             <AssistantPicker
               owner={owner}
-              assistants={agentConfigurations}
+              assistants={agentConfigurations.filter(
+                (agent) => agent.sId !== selectedAgentId
```

Comment:
You probably don't need this one because here `selectedAgent` is null which likely means that `selectedAgentId` is null which means that the filtering is a no-op but that's fine to keep as :+1: